packages/synapse-react-client/src/components/entity/page/title_bar/useGetEntityTitleBarProperties.tsx

```diff
@@ -84,9 +85,7 @@ export function useGetEntityTitleBarProperties(
     isVersionableEntity(bundle.entity) &&
     bundle.entity.isLatestVersion
   const { data: versionlessDOIAssociation } = useGetDOIAssociation(
-    entityId,
-    undefined,
-    'ENTITY',
+    { id: entityId, version: undefined, type: DoiObjectType.ENTITY },
```

Comment:
DOI hooks were updated to use the OpenAPI client, so I updated the `useGetDoiXxx` signatures to better match we pass to the client.